bitnami/victoriametrics/templates/vmagent/configmap.yaml

```diff
@@ -0,0 +1,347 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{/* Adding the helper in configmap.yaml for better readability */}}
+{{- define "victoriametrics.vmagent.defaultScrapeConfig" -}}
+global:
+  scrape_interval: 10s
+
+scrape_configs:
+  - job_name: vmagent
+    static_configs:
+      - targets: ["localhost:{{ .Values.vmagent.containerPorts.http }}"]
+
+    ## COPY from Prometheus helm chart https://github.com/helm/charts/blob/master/stable/prometheus/values.yaml
```

Comment:
Copy paste errror?

Reply:
No, it's directly copied from the upstream chart

https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-agent/values.yaml#L386